config-for-docker-build.js

```diff
@@ -11,6 +16,11 @@ exports.options = {
   'auto update': true,
   'session': true,
   'session store': 'mongo',
+  'session store options':{
+    autoRemoveInterval: process.env.KEYSTONE_SESSION_AUTO_REMOVE_INTERVAL || 10,
```

Comment:
If we set `autoRemoveInterval`, should `autoRemove` be set to `interval` rather than default value `native` ?

Reply:
Yes, I supposed I had a mistake here during the experiment. If we use `native`, the mongo will take care of the removal of the expired sessions through the TTL index. However, the expected behavior is to be controlled by our option. Thus, we need  toe provide outer interface for `autoRemove`